scripts/ci/build_droid.sh

```diff
@@ -52,11 +50,16 @@ if [ $AZURECLIDEV_ACR_SP_USERNAME ] && [ $AZURECLIDEV_ACR_SP_PASSWORD ]; then
 fi
 
 title 'Build docker image'
-docker build -t $image_name -f artifacts/Dockerfile.py36 artifacts
+docker build -t $image_name -f artifacts/Dockerfile artifacts
 
 title 'Push docker image'
 if [ "$1" == "push" ] || [ "$TRAVIS" == "true" ]; then
     docker push $image_name
 else
     echo "Skip"
 fi
+
+title 'Push docker image as latest'
+if [ "$TRAVIS" == "true" ]; then
+    docker tag $image_name azureclidev.azurecr.io/azurecli-test-$image_owner:latest
```

Comment:
Docker tag would not actually push it.
I assume the push happens elsewhere?

Reply:
You got me. Fixed in a newer commit.